### PR TITLE
Add Support for DragonWave Horizon

### PR DESCRIPTION
**Pull request by pheinrichs**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/horizon.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+os: horizon
+text: 'Dragonwave Horizon Compact'
+type: wireless
+icon: dragonwave
+over:
+    - { graph: device_wireless_snr, text: 'SNR' }
+    - { graph: device_wireless_power, text: 'Tx Power' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.7262.2.2
```
Comment (laf) [inline includes/definitions/horizon.yaml]: Looks like you can change that to just `1.3.6.1.4.1.7262.2.` to pick up more Horizon devices.

File: includes/definitions/horizon.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+os: horizon
```
Comment (laf) [inline includes/definitions/horizon.yaml]: Is this definitely the best os name? Is the firmware name?

File: includes/polling/os/horizon.inc.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * LibreNMS
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$version = trim(snmp_get($device, ".1.3.6.1.4.1.7262.2.2.1.3.2.1.0", "-Ovq"), '" ');
```
Comment (laf) [inline includes/polling/os/horizon.inc.php]: No mib available?

File: includes/definitions/horizon.yaml (hunk 2 above)
Comment (pheinrichs) [reply includes/definitions/horizon.yaml]: Horizon would be the best of name IMO.

File: includes/polling/os/horizon.inc.php (hunk 3 above)
Comment (pheinrichs) [reply includes/polling/os/horizon.inc.php]: I have yet to find an easy source of getting MIBS for dragonwave. I'll keep digging to see if I can get a source on them to add them.

**Comment by nosmowned** [comment]
Dragonwave Harmony Enhanced MC

[DW_HAR_MC_MIB.zip](https://github.com/librenms/librenms/files/1335125/DW_HAR_MC_MIB.zip)

**Pull request merged**